### PR TITLE
Add a test file as 1st commit

### PR DESCRIPTION
**Pull request by rdczowh**

### CODE REVIEW

**Comment by bpillalamarri** [comment]
👍

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+/bin/bash: q: command not found
```
Comment (bpillalamarri) [inline test.txt]: What's the weather like?

**Review by rdczowh — COMMENTED** [review]
very good try out

**Review by bpillalamarri — APPROVED** [review]
This is awesome work! 👍

**Pull request merged**